docs/rules/order.md

```diff
@@ -94,6 +94,32 @@ You can set the options like this:
 "import/order": ["error", {"groups": ["index", "sibling", "parent", "internal", "external", "builtin"]}]
 ```
 
+### `pathGroups: [array of objects]`:
+
+To be able so group by paths mostly needed with aliases pathGroups can be defined.
+
+Properties of the objects
+
+| property       | required | type   | description   |
+|----------------|:--------:|--------|---------------|  
+| pattern        |     x    | string | minimatch pattern for the paths to be in this group (will not be used for builtins or externals) |
+| patternOptions |          | object | options for minimatch, default: { nocomment: true } |
```

Comment:
there seem to be no tests using this option - can you add some? without them i have no idea why this is needed.

Reply:
Added a test using another option. I could also see to add this as default options and remove the options. But perhaps somebody has a usecase for negated pattern. So I though it would not hurt to give the possibility to adjust the options.